hi-ml-cpath/src/health_cpath/scripts/mount_azure_dataset.py

```diff
@@ -16,9 +16,9 @@
 from health_azure.utils import get_workspace  # noqa: E402
 
 
-def mount_dataset(dataset_id: str) -> Any:
+def mount_dataset(dataset_id: str, tmp_root: str = "/tmp/datasets/") -> Any:
```

Comment:
this script seems to be broken, to be fixed in separate PR
 ValueError: Unable to make dataset 'panda available for a local run because no AzureML workspace has been provided. Provide a workspace, or set a folder for local execution.to be fixed in a separate PR.

Reply:
Fixed